app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,186 @@ class SmokeTest {
             }
         }
     }
-}
+
+    @Test
+    fun verifyPageMainMenuItemsListInPortraitNormalMode() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {

```

Comment:
You don't need this, you can start with navigationToolbar ...

Reply:
Okay